package.json

```diff
@@ -15,6 +15,9 @@
     "url": "https://github.com/atlassian/react-beautiful-dnd/issues"
   },
   "main": "lib/index.js",
+  "files": [
+    "/lib"
```

Comment:
publishing this now?

Reply:
yeah - just the lib 💃 
I did a test run and it looks good

Reply:
oh this is because we got rid of the .npmignore?

Reply:
you can run `yarn pack` to see what will be published